force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -161,18 +200,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return The new string with the inputString repeated repeatCount times.
      */
     public static String repeatString(String inputString, Integer repeatCount) {
-        // Initialize the result String
-
-        // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
-            // If it is not the last iteration, add a semicolon to separate the strings
+        String resultingPhrase = inputString;
+        for (Integer i = 0; i < (repeatCount - 1); i++) {
+            resultingPhrase = resultingPhrase + '; ' + inputString;
         }
-        */
-
-        // Return the final result string
-        return null; // Replace null with the variable you used to store the result
+        return resultingPhrase;
     }
```

Comment:
This was a tricky problem with a lot of ways to solve it. Good work here. 

The challenge is, can you do it on one line? Research the methods on the String class